autoware_lanelet2_extension/test/src/test_projector.cpp

```diff
@@ -32,52 +32,40 @@ class TestSuite : public ::testing::Test  // NOLINT for gtest
 TEST(TestSuite, ForwardMGRSProjection)  // NOLINT for gtest
 {
   lanelet::projection::MGRSProjector projector;
-  // lat/lon in Tokyo
-  lanelet::GPSPoint gps_point;
-  gps_point.lat = 35.652832;
-  gps_point.lon = 139.839478;
-  gps_point.ele = 12.3456789;
-  lanelet::BasicPoint3d mgrs_point = projector.forward(gps_point);
-
-  // projected z value should not change
-  ASSERT_DOUBLE_EQ(mgrs_point.z(), gps_point.ele)
-    << "Forward projected z value should be " << gps_point.ele;
-
-  // https://www.movable-type.co.uk/scripts/latlong-utm-mgrs.html
-  // round the projected value to mm since the above reference only gives value
-  // in mm precision
-  ASSERT_EQ(projector.getProjectedMGRSGrid(), "54SUE") << "Projected grid should be "
```

Comment:
This test is supposed to compare the return value of `getProjectedMGRSGrid`, which seems to be lost with the fixes in this PR.